app/views/browse/_common_details.html.erb

```diff
@@ -31,4 +31,4 @@
 </div>
 <% end %>
 
-<%= render :partial => "tag_details", :object => common_details.tags %>
+<%= render :partial => "tag_details", :object => (local_assigns[:tag_history] or common_details.tags), :locals => { :is_history => local_assigns[:is_history] } %>
```

Comment:
`local_assigns` is needed because these variables aren't *always* passed (I think?), and so referring to them directly will throw an exception when they are missing, rather than just returning `nil`. I suspect there's a better way to do this, though!

Reply:
Fixed this by ensuring the two variables are always set. Sorry for missing this at first!